src/app/api-server/src/app.rs

```diff
@@ -55,15 +63,16 @@ pub async fn run(matches: clap::ArgMatches, config: ApiServerConfig) -> Result<(
         repo_url = %repo_url,
         local_dir = %local_dir.path().display(),
         config = ?config,
-        "Initializing {}",
-        BINARY_NAME
+        "Initializing {BINARY_NAME}",
     );
 
+    let logged_account_subject = CurrentAccountSubject::logged(
+        AccountID::new_seeded_ed25519(DEFAULT_ACCOUNT_NAME.as_bytes()),
+        DEFAULT_ACCOUNT_NAME.clone(),
```

Comment:
Let's avoid binding to CLI's `DEFAULT_ACCOUNT_NAME`, we are going to remove it after solving a couple problems. Instead, there are a few possibilities:
 a) Skip deep digging for now and simply define a local constant for `kamu` account
 b) Auto-derive the name of the user from OS user name (see `kamu-cli`, `AccountService` does something like that for multi-tenant repos). The version deployed in the cluster would get some name from the assigned service account's name.
 c) have a way to specify the name of the active user in the command  line / configuration.

Reply:
Normally, this should be the OS user. We should not be implicitly assuming `kamu` account.

Reply:
a) Local constant for now